raiden-ts/src/services/actions.ts

```diff
@@ -45,3 +45,26 @@ export interface iouClear extends ActionType<typeof iouClear> {}
 
 export const udcDeposited = createAction('udc/deposited', UInt(32));
 export interface udcDeposited extends ActionType<typeof udcDeposited> {}
+
+export const udcWithdraw = createAsyncAction(
+  t.partial({}),
+  'udc/withdraw/request',
+  'udc/withdraw/success',
+  'udc/withdraw/failure',
+  t.type({ amount: UInt(32) }),
+  t.type({ txHash: Hash, amount: UInt(32) }),
```

Comment:
Is this a `ConfirmableAction`? It be good to stick to the schema (txHash, txBlock, confirmed)

Reply:
it is kind of weird, not really a confirmable action I just wait inside before I fire the withdrawPlanned